src/main/scala/gestalt/Toolbox.scala

```diff
@@ -263,6 +263,16 @@ trait TypeToolbox extends Toolbox { t =>
   def <:<(tp1: Type, tp2: Type): Boolean
   def typeOf(path: String): Type
 
+  val Ascribe: AscribeHelper
+  trait AscribeHelper {
+    def unapply(arg: Tree): Option[(Tree, TypeTree)]
+  }
+
+  val SeqLiteral: SeqLiteralHelper
+  trait SeqLiteralHelper {
+    def unapply(tree: Tree): Option[Seq[Tree]]
+  }
```

Comment:
It is possible to transform it to a method call on one of the collections. I tried that before with `Seq`.The cleaned up version is at valdisxp1/gestalt@48efff5
The main component is this special case in Apply extractor:
```scala
case c.SeqLiteral(items, elemtpt) => Some((c.Select(c.TypeApply(c.Ident("Seq".toTermName), List(elemtpt)), "apply".toTermName), items))
```